src/types.ts

```diff
@@ -134,6 +134,7 @@ export interface FormField {
   questionInfo?: string;
   historicalExpression?: string;
   constrainMaxWidth?: boolean;
+  inlineMultiCheckbox?: boolean;
```

Comment:
Minor comment: `useCheckboxGroup` reads better

Reply:
Good alternative but I went with this naming because it was more intuitive to it's purpose which was displaying them in a line.